requirements.txt

```diff
@@ -1,5 +1,23 @@
+appnope==0.1.0
```

Comment:
Yeah. I think I did an accidental freeze that grabbed stuff from the virtualenv that we didn't need. I closed that pull request w/out merging.